server.go

```diff
@@ -449,3 +449,18 @@ func WithLoggingMiddleware(loggingConfig LoggingConfig) func(*Server) {
 		}
 	}
 }
+
+// WithStripTrailingSlash enables stripping of trailing slashes from routes and requests
+// Default is false.
+//
+// Example:
+//
+//	app := fuego.NewServer(
+//		fuego.WithStripTrailingSlash(),
+//	)
+func WithStripTrailingSlash() func(*Server) {
+	return func(s *Server) {
+		// Add OptionStripTrailingSlash to the default route options
+		s.routeOptions = append(s.routeOptions, OptionStripTrailingSlash())
+	}
+}
```

Comment:
Very good! With one Server Option, they will be able to have this feature :)

I think it should look like this:

```go
s.routeOptions = append(s.routeOptions, OptionStripTrailingSlash())
s.globalMiddlewares = append(s.globalMiddlewares, stripTrailingSlashesMiddleware)
```

Reply:
Yeah I'm a bit confused here as well. Why on the globalMiddlewares? That seems more like a thing for cors? At least with just 

```
s.routeOptions = append(s.routeOptions, OptionStripTrailingSlash())
```

You can apply this at both the server and group level

Reply:
Ope nevermind. I see now this literally isn't possible unless you register stripping the path at the global level. Got it. I'll make a PR on how I think this should look.